meilisearch/models/client.py

```diff
@@ -0,0 +1,64 @@
+from datetime import datetime
+from typing import List, Optional
+
+from camel_converter.pydantic_base import CamelBase
+
+
+class _KeyBase(CamelBase):
+    uid: str
+    name: Optional[str] = None
+    description: str
+    actions: List[str]
+    indexes: List[str]
+    expires_at: Optional[datetime] = None
+
+    class Config:
+        json_encoders = {
+            datetime: lambda v: None
+            if not v
+            else f"{str(v).split('.', maxsplit=1)[0].replace(' ', 'T')}Z"
+        }
+
+
+class Key(_KeyBase):
+    key: str
+    created_at: datetime
+    updated_at: Optional[datetime] = None
+
+
+class KeyCreate(CamelBase):
+    name: Optional[str] = None
+    description: str
+    actions: List[str]
+    indexes: List[str]
+    expires_at: Optional[datetime] = None
+
+    class Config:
+        json_encoders = {
+            datetime: lambda v: None
+            if not v
+            else f"{str(v).split('.', maxsplit=1)[0].replace(' ', 'T')}Z"
+        }
+
+
+class KeyUpdate(CamelBase):
+    key: str
+    name: Optional[str] = None
+    description: Optional[str] = None
+    actions: Optional[List[str]] = None
+    indexes: Optional[List[str]] = None
+    expires_at: Optional[datetime] = None
+
+    class Config:
+        json_encoders = {
+            datetime: lambda v: None
+            if not v
+            else f"{str(v).split('.', maxsplit=1)[0].replace(' ', 'T')}Z"
+        }
+
+
+class KeySearch(CamelBase):
```

Comment:
You couldn't know it is a tacit agreement within the team but for reasons of consistency between SDKs in v0.28, we decided to call all structures or classes that have results to name them as `ObjectsResults` so this one should be named `KeysResults`.